srcs/incs/HttpStatusCodes.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef SRCS_INCS_HTTPSTATUSCODES_HPP_
+#define SRCS_INCS_HTTPSTATUSCODES_HPP_
+#include <map>
+#include <string>
+
+class HttpStatusCodes {
+	private:
+		typedef std::size_t								StatusCode_;
```

Comment:
No entiendo para qué hacen falta estos dos typedef.

Un typedef se supone que sirve para condensar un tipo complejo en menos palabras, a través de un alias, pero un typedef de size_t o un typedef de std::string, desde mi punto de vista, no tiene sentido. Si tienes que explicarlo y dejar constancia, un comentario vale igual, y no deja código muerto. 

Ni siquiera lo usas en la implementación de la funcion que podría usarlo (const std::map<std::size_t, std::string>
HttpStatusCodes::CreateStatusCodesMap_()

Reply:
I removed the typedefs and added a comment instead. I don't mind if it is a comment or typedefs, as long as it improves readability.